rowset.ts

```diff
@@ -0,0 +1,13 @@
+import { inspect } from 'util'
+import row from './row'
+import S from './sanitised'
+
+export const rowset =
+  <T>(xs: T[]): S => {
+    if (!Array.isArray(xs)) {
+      throw new TypeError(`Expected array, got ${inspect(xs)}.`)
+    }
+    return row<any>(xs, row)
```

Comment:
If you want, this can be a second generic so that the caller also gets the correct types on the return

Reply:
I'm not sure I follow what you mean?